distribution/scripts/ei/microei-start.sh

```diff
@@ -94,11 +104,22 @@ capp_dir=$script_dir/capp/
 echo "Starting the docker container:"
 (
     set -x
-    docker run --name=microei -d -p 8280:8290 -p 8243:8253 --add-host=netty:$netty_host --cpus=${cpus} \
-        --volume $(realpath $capp_dir):/home/wso2carbon/wso2ei-${wso2_ei_version}/wso2/micro-integrator/repository/deployment/server/carbonapps \
-        --volume ${HOME}/logs/wso2carbon.log:/home/wso2carbon/wso2ei-${wso2_ei_version}/wso2/micro-integrator/repository/logs/wso2carbon.log \
-        --volume ${HOME}/logs/gc.log:/home/wso2carbon/wso2ei-${wso2_ei_version}/wso2/micro-integrator/repository/logs/gc.log \
-        -e "${JVM_MEM_OPTS}" -e "${JAVA_OPTS}" wso2ei-micro-integrator:${wso2_ei_version}
+    if test "$server_type" = "mei"
```

Comment:
Do not use `test` command here. Use `[[`. For example:  `if [[ $server_type == "mei" ]]; then`

Reply:
Corrected